test/action.yml

```diff
@@ -8,11 +8,11 @@ runs:
       shell: bash
       working-directory: ${{ env.DIRECTORY_PATH }}
     - run: $PACKAGE_MANAGER install --ignore-scripts
-      shell: bash
+      shell: bash -l {0}
```

Comment:
Is it necessary to invoke bash as login shell?

Reply:
It is, because otherwise `nvm` is never activated and the system Node is used instead, which will always be the version provided by GitHub Actions which is not what we want.